packages/pds/src/db/tables/repo-root.ts

```diff
@@ -5,6 +5,7 @@ export interface RepoRoot {
   rev: string | null
   indexedAt: string
   takedownId: number | null
+  emailConfirmedAt: string | null
```

Comment:
I believe this column comes from the `user_account` table rather than the `repo_root` table.